coralproject/talk
# [CORL-2395]: story title on single comment view card

## What does this PR do?

These changes ensure that the story title and `Moderate story` link appear on the moderate cards for the single comment view. They also migrate tests for `singleComment.spec.tsx` to RTL. They also remove some renders correctly tests for the single comment view.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Navigate to a single comment view (can go to a moderate queue and, on any comment, click `Details` --> `In Moderation` then paste the link into a new tab). See that the story title and `Moderate story` link are there. 

Also check that the story title and `Moderate story` link still appear on comment cards in moderate queues when filtering by all stories. Check that the story title and `Moderate story` link do **not** appear on comment cards in moderate queues when filtering by a specific story.
 
## How do we deploy this PR?
